.github/workflows/continuous-integration.yml

```diff
@@ -147,7 +147,7 @@ jobs:
 
           MISSED_TRANSLATIONS=$( echo "$ORIGINAL_ARTICLES" | tr \\n \\0 | xargs -0 dirname | sort -u | {
             while read DIRECTORY; do
-              for ARTICLE_NAME in $( ls "$DIRECTORY"/??.md | grep -v en.md || true ); do
+              for ARTICLE_NAME in $( ls "$DIRECTORY"/(??|??-??).md | grep -v en.md || true ); do
```

Comment:
this would work

![image](https://user-images.githubusercontent.com/36758269/160256861-4ec3b047-3c27-48a2-bc41-b1c82add694a.png)

```suggestion
              for ARTICLE_NAME in $( ls "$DIRECTORY"/??.md "$DIRECTORY"/??-??.md | grep -v en.md || true ); do
```

Reply:
it fails when there aren't any `??-??` files. I'll attempt something else later